lib/rubocop/minitest/assert_offense.rb

```diff
@@ -105,8 +107,7 @@ def assert_no_offenses(source, file = nil)
 
       def assert_offense(source, file = nil, **replacements)
         setup_assertion
-
-        @cop.instance_variable_get(:@options)[:autocorrect] = true
+        enable_autocorrect
```

Comment:
I was forced to make this change because changing the instance variable to a method call  caused method to have an `AbcSize` offense.